src/components/ImageBox.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Main.css';
+import Title from './Title';
+import kittenImage from '../assets/kitten1.jpeg';
+
+const ImageBox = (props) => (
+  <div className={styles.ImageBox}>
+    <img src={kittenImage} alt={props.name} />
+    <Title id={props.id} name={props.name}/>
+  </div>
+);
+
+ImageBox.propTypes = {
+  id: PropTypes.number,
```

Comment:
I'm curious if id isn't maybe required, or should be required? I see multiple PropTypes that aren't required, but it seems like some of them actually might be, in the future-- just a thought :)